src/views/SystemProfile/index.tsx

```diff
@@ -310,6 +311,19 @@ export function showVal(
   return val;
 }
 
+/**
+ * Determine whether a particular sub page is editable based on sub page key.
+ */
+export function subPageIsEditable(componentID: SubpageKey): boolean {
+  // TODO: Add sub pages as they become editable (this will remove the temporary edit system profile banner from that sub page)
+  const editableComponentIDs = ['home', 'team']; // NOTE: this list is based off of the SubpageKey list in types/systemProfile
```

Comment:
I thought about pulling this list of editable sub pages out and storing it somewhere but I wasn't sure if there was a good place to put it since it should temporary. I considered types/systemProfile among others, thoughts?

Reply:
In terms of how the line reads, maybe
`const editableComponentIDs: SubpageKey[] = ['home', 'team'];`
could be more self-explanatory with types and not need the line note anymore

The other comments are good